### DISCUSSION TITLE
Public page inside private page tree

### DISCUSSION THREAD

**Discussion by matteotognini** [wagtail/wagtail#12654] [Ideas]
### Is your proposal related to a problem?
As explained in the documentation [here ](https://docs.wagtail.org/en/stable/topics/permissions.html#page-permissions)it is currently not possible to set differents permissions for pages within a private page tree. All pages inside a private tree inherit the permissions of the parent private page.

### Describe the solution you'd like
In some cases specific child pages might need to be accessible to certain user groups or individuals, while others remain restricted
Currently, this use case cannot be accommodated due to inherited permissions across the tree.
 

Could this feature be considered for a future release? What is your opinion on this feature? Do you think it would be useful?

**Comment by gasman** [comment]
Hi @matteonext - thanks for the suggestion.

I'll move this to a discussion for now, as we'd really need to get some consensus about how such a feature would behave and how it can be made as useful as possible to the most users, before it's possible to be picked up as an issue.

Opinions welcome. Perhaps this calls for a new option in the 'set page privacy' dialog, to be shown as a checkbox whenever a restriction already exists on the parent, something like:
![Image](https://github.com/user-attachments/assets/d264e08c-4926-436f-afef-4a1a2de898e6)
